packages/genesys-spark-components/src/components/beta/gux-time-zone/gux-time-zone.tsx

```diff
@@ -0,0 +1,72 @@
+import { Component, JSX, h, Host, Prop, Element } from '@stencil/core';
+import { trackComponent } from '@utils/tracking/usage';
+import { buildI18nForComponent, GetI18nValue } from '../../../i18n';
+import translationResources from '../gux-time-zone-picker/i18n/en.json';
+import { GuxTimeZoneIdentifier } from '../../../i18n/time-zone/types';
+
+import { getLocalizedOffset, shortenZone } from './gux-time-zone.service';
+
+@Component({
+  tag: 'gux-time-zone-beta',
+  shadow: true
+})
+export class GuxTimeZoneBeta {
+  private i18n: GetI18nValue;
+
+  /**
+   * Reference to the host element.
+   */
+  @Element()
+  root: HTMLElement;
+
+  /**
+   * The id of the time zone to display
+   */
+  @Prop()
+  timeZoneId: GuxTimeZoneIdentifier;
+
+  /**
+   * True to display the zone's offset from UTC
+   */
+  @Prop()
+  offset: boolean;
+
+  /**
+   * True to display surround the offset with parentheses
```

Comment:
This phrasing is confusing to me: `True to display surround the offset with parentheses`